src/pages/developers/tutorials.tsx

```diff
@@ -431,6 +507,7 @@ const TutorialsPage = ({ data, pageContext }) => {
         )}
         {state.filteredTutorials.map((tutorial) => {
           return (
+            // Todo: Generate a unique id for each tutorial, to property (youtube url) is duplicated in some tutorials
             <TutorialCard key={tutorial.to} to={tutorial.to} hideArrow>
```

Comment:
React actually complains here because of a duplicated key. To property represents the Hyperlink URL of the tutorial but seems like in some cases that url is duplicated. Left a todo just in case